skimage/segmentation/boundaries.py

```diff
@@ -199,6 +212,8 @@ def mark_boundaries(image, label_img, color=(1, 1, 0),
     --------
     find_boundaries
     """
+    if image.dtype == 'bool':
+        image = image.astype(np.uint8)
```

Comment:
This isn't needed. `find_boundaries` will take care of bool images where necessary.